contrib/babelfishpg_tsql/src/pltsql_ruleutils.c

```diff
@@ -1929,11 +2276,15 @@ quote_identifier(const char *ident)
 		}
 	}
 
-	if (quote_all_identifiers)
+         if (quote_all_identifiers)
 		safe = false;
 
 	if (safe)
 	{
+	
+
+
+	
```

Comment:
Please fix unwanted whitespace and indent.

Reply:
done!